src/Concordium/Client/Runner.hs

```diff
@@ -1385,15 +1430,13 @@ startTransaction txCfg pl confirmNonce maybeAccKeys = do
   nonce <- getNonce naAddr n confirmNonce
   accountKeyMap <- case maybeAccKeys of
                      Just acKeys' -> return acKeys'
-                     Nothing -> liftIO $ failOnError $ decryptAccountKeyMapInteractive esdKeys (Nothing) Nothing
+                     Nothing -> liftIO $ failOnError $ decryptAccountKeyMapInteractive esdKeys Nothing Nothing
   let sender = applyAlias tcAlias naAddr
   let tx = signEncodedTransaction pl sender energy nonce expiry accountKeyMap
   when (isJust tcAlias) $
       logInfo [[i|Using the alias #{sender} as the sender of the transaction instead of #{naAddr}.|]]
-  sendTransactionToBaker tx defaultNetId >>= \case
-    Left err -> fail err
-    Right (GRPCResponse _ False) -> fail "transaction not accepted by the baker"
```

Comment:
This is changed. Is it bad to rely on the node response text from the API in this case?

Reply:
No, but I would still state something like
```
transaction not accepted by the baker: {error from the node}
```
to provide context for the user.